Sources/Services/AWSGroundStation/Sources/AWSGroundStation/Models.swift

```diff
@@ -5474,7 +5474,7 @@ extension GroundStationClientTypes.SpectrumConfig {
         var value = GroundStationClientTypes.SpectrumConfig()
         value.centerFrequency = try reader["centerFrequency"].readIfPresent(with: GroundStationClientTypes.Frequency.read(from:))
         value.bandwidth = try reader["bandwidth"].readIfPresent(with: GroundStationClientTypes.FrequencyBandwidth.read(from:))
-        value.polarization = try reader["polarization"].readIfPresent() ?? .`none`
+        value.polarization = try reader["polarization"].readIfPresent() ?? GroundStationClientTypes.Polarization.`none`
```

Comment:
This is the specific enum default value that was ambiguous, because the compiler couldn't tell if `.none` was `Optional.none` or `GroundStationClientTypes.Polarization.none`.